extensions/mssql/src/objectManagement/objectManagementService.ts

```diff
@@ -432,6 +434,27 @@ export class TestObjectManagementService implements IObjectManagementService {
 		};
 	}
 
+	private getDatabaseView(isNewObject: boolean, name: string): DatabaseViewInfo {
+		return isNewObject ? <DatabaseViewInfo>{
```

Comment:
Shouldn't these have some appropriate defaults? Or are you planning on adding that later?